minlib.js

```diff
@@ -1,46 +1,28 @@
-(function () {
-  "use strict";
-  
-  // querySelector
-
-  const sel = (selectors, baseElement = document) => {  
-    return baseElement.querySelector(selectors);
-  };
-
-  // querySelectorAll
-
-  const all = (selectors, baseElement = document) => {
-      return [...baseElement.querySelectorAll(selectors)];
-  };
-
-  // addEventListener
-
-  const addEvt = (target, type, listener) => {
-
-    if (Array.isArray(target) || typeof target === 'string'){ 
-      if(!Array.isArray(target)){
-        target = all(target);
-      };
-      for(let element of target){
-        element.addEventListener(type, listener);
-      };
-    } else {
-      target.addEventListener(type, listener);
-    };
-
-  };
-
-  // DOMContentLoaded
-
-  const loaded = (listener) => {
-    addEvt(document, 'DOMContentLoaded', listener);
-  };
-  
-  // export
-  
-  window.sel = sel;
-  window.all = all;
-  window.addEvt = addEvt;
-  window.loaded = loaded;
-  
-}());
\ No newline at end of file
+window.query = (selector, baseElement = document) => {
+  return baseElement.querySelector(selector);
+};
+
+// querySelectorAll
+window.queryAll = (selector, baseElement = document) => {
+  if (typeof selector === 'string') {
+    return [...baseElement.querySelectorAll(selector)];
+  }
+
+  if (selectors.length) {
+    return [...selector];
+  }
+
+  return [selector];
+};
+
+// addEventListener
+window.addEvent = (targets, type, listener, useCapture = false) => {
+  queryAll(targets).forEach(target => {
+    target.addEventListener(type, listener, useCapture);
+  });
+};
+
+// DOMContentLoaded
+window.loaded = (listener) => {
+  addEvent(document, 'load', listener);
```

Comment:
> So you assume how users would use your script

I assume how users would use my code, because there is a documentation for this method. 

> you admit that reliabilty of jQuery is based on fact that it doesn't assume

I don't - in this case jQ assumes, that your code should be in `$( document ).ready` callback.